src/openfermion/tests/_example_test.py

```diff
@@ -18,24 +18,52 @@
 import tempfile
 import unittest
 
-import nbformat
 import numpy
+import pytest
 
 from openfermion.config import THIS_DIRECTORY
 
 
+def _module_import(plug):
```

Comment:
As you asked about in your PR, I think we can put this _module_import helper function into utils/testing_utils.py

Reply:
Also, for possible future users of this function, can you add a docstring and specification for the plug parameter?